lib/logitech_receiver/receiver.py

```diff
@@ -133,6 +137,12 @@ def __init__(self, receiver, number, link_notification=None):
         # also it gets set to None on this object when the device is unpaired
         assert self.wpid is not None, 'failed to read wpid: device %d of %s' % (number, receiver)
 
+        for dev in _hid.enumerate({'vendor_id': 0x046d}):
```

Comment:
This fails on my system because the device hidraw nodes are not accessible by regular users.   Solaar has a udev rule that makes the receiver nodes accessible but not the device nodes.   
@FFY00 Do you have any ideas on how the udev rule could be extended to make these nodes accessible?

Reply:
I *think* I have a solution, and it is simple.  Just remove this part of the udev rule:
```
# don't apply to the paired peripherals, just the receivers
DRIVERS=="logitech-djdevice|logitech-hidpp-device", GOTO="solaar_end"

```

Reply:
The wired devices will have to be added to the rule, though, but I think that is fine.